src/coreclr/src/jit/gentree.h

```diff
@@ -775,16 +775,24 @@ struct GenTree
 #define GTF_VAR_CLONED      0x01000000 // GT_LCL_VAR -- this node has been cloned or is a clone
                                        // Relevant for inlining optimizations (see fgInlinePrependStatements)
 
-// TODO-Cleanup: Currently, GTF_REG_BIRTH is used only by stackfp
-//         We should consider using it more generally for VAR_BIRTH, instead of
-//         GTF_VAR_DEF && !GTF_VAR_USEASG
-#define GTF_REG_BIRTH       0x04000000 // GT_LCL_VAR, -- enregistered variable born here
-#define GTF_VAR_DEATH       0x02000000 // GT_LCL_VAR, -- variable dies here (last use)
-
 #define GTF_VAR_ARR_INDEX   0x00000020 // The variable is part of (the index portion of) an array index expression.
                                        // Shares a value with GTF_REVERSE_OPS, which is meaningless for local var.
-
-#define GTF_LIVENESS_MASK (GTF_VAR_DEF | GTF_VAR_USEASG | GTF_REG_BIRTH | GTF_VAR_DEATH)
+#define GTF_VAR_MULTIREG    0x20000000 // This is a struct or (on 32-bit platforms) long variable that is used or defined
+                                       // to/from a multireg source or destination (e.g. a call arg or return, or an op
+                                       // that returns its result in multiple registers such as a long multiply).
+// Last-use bits for a MULTIREG var. Note that a node marked GTF_VAR_MULTIREG can only be a pure
+// definition of all the fields, or a pure use of all the fields, so we don't need the equivalent of
+// GTF_VAR_USEASG.
+#define GTF_VAR_MULTIREG_DEATH0 0x00020000 // GT_LCL_VAR -- The last-use bit for a lclVar (the first register if it is multireg).
+#define GTF_VAR_DEATH           GTF_VAR_MULTIREG_DEATH0
+#define GTF_VAR_MULTIREG_DEATH1 0x00040000 // GT_LCL_VAR -- The last-use bit for the second register of a multireg lclVar.
+#define GTF_VAR_MULTIREG_DEATH2 0x00080000 // GT_LCL_VAR -- The last-use bit for the third register of a multireg lclVar.
+#define GTF_VAR_MULTIREG_DEATH3 0x00100000 // GT_LCL_VAR -- The last-use bit for the fourth register of a multireg lclVar.
```

Comment:
I'm not sure that you can use these bits to track the MULTIREG_DEATH info.

Only the upper 8-bits are node specific , 
the lower 24 bits are common for all nodes:  (see GTF_COMMON_MASK)
If a particular node needs more that 8 bits of flags, then another approach has to be taken.

Reply:
Actually, there are unused bits sprinkled about. I will restructure these to ensure that I'm using bits that aren't used by other nodes.

Reply:
I need a flag bit on LCL_VAR too, see #34827. Hopefully enough to go around.

Reply:
I've now consolidated the `GTF_VAR` flags in bits `0xffc00000`, and the `GTF_COMMON_MASK` is `0x0003FFFF` after consolidating. So by my reckoning we have 4 bits in `0x003c0000` that are now available.